docs/tasks/federation/policy.rego

```diff
@@ -0,0 +1,58 @@
+# OPA supports a high-level declarative language named Rego for authoring and
+# enforcing policies. For more infomration on Rego, visit
+# http://openpolicyagent.org.
+
+# Rego policies are namespaced by the "package" directive.
+package kubernetes.placement
+
+# Imports provide aliases for data inside the policy engine. In this case, the
+# policy simply refers to "clusters" below.
+import data.kubernetes.clusters
+
+# The "annotations" rule generates a JSON object containing the key
+# "federation.kubernetes.io/replica-set-preferences" mapped to <preferences>.
+# The preferences values is generated dynamically by OPA when it evaluates the
+# rule.
+#
+# The SchedulingPolicy Admission Controller running inside the Federation API
+# server will merge these annotatiosn into incoming Federated resources. By
+# setting replica-set-preferences, we can control the placement of Federated
```

Comment:
IIUC, this will also work with cluster-selectors?
If yes, then please add an example for that as well.
If not, we should be explicit that this works only for replicasets.